internal/alpini/netty4/alpini-netty4-base/src/test/java/com/linkedin/alpini/util/TestNettyUtil.java

```diff
@@ -0,0 +1,23 @@
+package com.linkedin.alpini.util;
+
+import java.util.function.Supplier;
+import org.testng.SkipException;
+
+
+public class TestNettyUtil {
+  public static <T> T skipEpollIfNotFound(Supplier<T> eventLoopGroupSupplier) {
+    try {
+      return eventLoopGroupSupplier.get();
+    } catch (NoClassDefFoundError e) {
+      if (e.getMessage().equals("Could not initialize class io.netty.channel.epoll.EpollEventLoop")) {
+        throw new SkipException("EPoll not available.");
+      }
+      throw e;
```

Comment:
Here and in the next catch, do we expect to specified exception with different messages? or its just for safety ?
nit: since we are doing similar action can we catch using `catch (NoClassDefFoundError | UnsatisfiedLinkError e)` ?

Reply:
I wanted to verify the error message so that the catch is as specific as possible (even more specific than just the exception type) in order to let any other kind of issue bubble up.

We could use the pipe notation, but I would still want to check the exception messages, so I don't think it would reduce the code size much...